packages/medusa-core-utils/src/index.ts

```diff
@@ -12,3 +12,4 @@ export * from "./medusa-container"
 export { parseCorsOrigins } from "./parse-cors-origins"
 export { transformIdableFields } from "./transform-idable-fields"
 export { default as zeroDecimalCurrencies } from "./zero-decimal-currencies"
+export { createMedusaContainer } from "./medusa-container"
```

Comment:
thought(non-blocking): Should we move it to `@medusajs/utils` while we are at it? 😜

Reply:
@adrien2p We can also merge this now, if it is blocking and then do the move in a subsequent PR. What do you think?

Reply:
at line 11 we have `export * from "./medusa-container"`

Reply:
No it is exported in fact, it was just my IDE playing with me 😂

Reply:
So I just do the move to utils/types only